### PR TITLE
Fix document formatting errors

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
Fix document formatting errors

image

image

### CODE REVIEW

**Comment by ZP-AlwaysWin** [comment]
/assign @markthink
Please help me to review ，Thanks

**Comment by idealhack** [comment]
/lgtm

thanks

File: content/zh/docs/concepts/workloads/controllers/garbage-collection.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Here's a configuration file for a ReplicaSet that has three Pods:
 ## 所有者和附属
 
 某些 Kubernetes 对象是其它一些对象的所有者。例如，一个 ReplicaSet 是一组 Pod 的所有者。
-具有所有者的对象被称为是所有者的*附属*。
+具有所有者的对象被称为是所有者的 *附属* 。
 每个附属对象具有一个指向其所属对象的 `metadata.ownerReferences` 字段。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/workloads/controllers/garbage-collection.md:54]: nice job!
Thanks
/lgtm

**Pull request merged**